<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_addToClass</class>
 <widget class="QDialog" name="Dialog_addToClass">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>105</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>219</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Categorys</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../_internal/Static/link.ico</normaloff>../_internal/Static/link.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
                background-color: rgb(74, 74, 74);
                color: #fff
                }
                QPushButton:hover {
                background-color: rgb(47, 47, 47);
                }

                QComboBox:hover {
                background-color: rgb(47, 47, 47);
                }
                QComboBox{
                border: 1px solid rgb(195, 195, 195);
                background-color: rgb(74, 74, 74);
                border-radius: 5px;
                padding:5px;
                }
                QComboBox QAbstractItemView {
                background-color: rgb(47, 47, 47);
                border: none;
                border: 1px solid rgb(189, 189, 189);
                border-radius: 5px;
                padding:5px;
                }

                QComboBox::drop-down{
                background-color: rgb(47, 47, 47);
                border-radius: 5px;
                padding:5px;
                }

                #button_addClass{
                border: 1px solid rgb(14, 172, 51);
                border-radius: 5px;
                padding:8px;
                }
                #button_removeClass{
                border: 1px solid rgb(213, 50, 22);
                border-radius: 5px;
                padding:9px;
                }

                #button_save{
                border: 1px solid rgb(14, 172, 51);
                border-radius: 5px;
                padding:8px;
                }
                #button_cancel{
                border: 1px solid rgb(213, 50, 22);
                border-radius: 5px;
                padding:8px;
                }
            </string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QComboBox" name="class_items">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="button_save">
           <property name="text">
            <string>save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_cancel">
           <property name="text">
            <string>cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
